### DISCUSSION TITLE
Lesson 5: Compiling SimpleStorage.sol with brownie

### DISCUSSION THREAD

**Discussion by Rogelio1997** [smartcontractkit/full-blockchain-solidity-course-py#623] [Q&A] [answered]
When I compile SimpleStorage.sol I get a "Project has been compiled" message but the SimpleStorage.json doesn't appear. I tried to continue with the course but later when running the script deploy.py I get "ImportError: cannot import name 'SimpleStorage' from 'brownie', How can I solve this? 

I know someone had the same problem in "https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/185" but I am not using a virtual environment so I am not sure how I should proceed. 

![Screenshot from 2021-12-23 13-58-31](https://user-images.githubusercontent.com/62615214/147286694-5407593d-68a9-43ac-ba15-505a374147eb.png)

When compiling with brownie 

![image](https://user-images.githubusercontent.com/62615214/147289720-d7ab09b8-58f9-475a-92f8-1638ed197a23.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Rogelio1997 using virtual envs is the best option to manage python projects, you can install virtualenv trough pip.
Also installing brownie with pipx is not recommended for linux users.
If you want a more straight forward guide about how to configure tour windows machine to work with web3, please refer to the course index apart "windows support":

**Reply by Rogelio1997** [reply]
It works now using venv, thanks

**Comment by 0xFLT** [comment]
That other answer seems unnecessary.
For me, the answer was to use the compiled contract file from Browniefolder>Build>Contracts>FileName.json instead of the .sol file name.
